BlocSpot/app/src/main/java/com/izor066/android/blocspot/api/model/PointsOfInterest.java

```diff
@@ -0,0 +1,41 @@
+package com.izor066.android.blocspot.api.model;
+
+/**
+ * Created by igor on 22/9/15.
+ */
+public class PointsOfInterest {
```

Comment:
I would incline to name this `PointOfInterest` rather then Point**s**. The plural often means that the class represents a collection of items, but in this case it doesn't, it's just one point of interest.